arangod/Pregel/Conductor.cpp

```diff
@@ -455,12 +455,10 @@ static void resolveInfo(
 }
 
 /// should cause workers to start a new execution
-ErrorCode Conductor::_initializeWorkers(std::string const& suffix,
-                                        VPackSlice additional) {
+auto Conductor::_initializeWorkers(VPackSlice additional)
+    -> std::vector<futures::Future<ResultT<GraphLoaded>>> {
```

Comment:
Could this somehow be a single future which just waits for all the insidey ones to resolve?

Reply:
Good idea, I now return `futures::Future<std::vector<
        futures::Try<arangodb::ResultT<arangodb::pregel::GraphLoaded>>>>`

Reply:
Not for this PR: 

In the next iteration this could do with a `using GraphLoadedFuture = std::vector<futures::Future<ResultT<GraphLoaded>>>` (and then I get ideas about how this should be done abstractly for a collection of workers.)